Week-4/OOP_js/Practice_ES6/script.js

```diff
@@ -0,0 +1,122 @@
+//Session-1 Inheritance
+class Vehicle {
+  constructor(Make, Model, Year, Color) {
+    this.make = Make;
+    this.model = Model;
+    this.year = Year;
+    this.color = Color;
+  }
```

Comment:
constructor args need not be capitalized

Reply:
Fine